### DISCUSSION TITLE
WebREPL client status?

### DISCUSSION THREAD

**Discussion by PBrunot** [micropython/micropython#10284] [General]
Hi,
The web repl seems abandonned https://github.com/micropython/webrepl even if there are interesting PR.
Is there any plan / updates coming on ?

The reason I am asking is that using a smartphone while debugging micropython device can be helpful, but the actual webrepl client  is not usable on my android phone. On PC/Windows I am using WebREPL client in Thonny and does not feel the same need.

**Comment by jimmo** [comment]
@PBrunot sorry about the delay responding.

It's not abandoned, just hasn't had any major updates (other than the fixes I did a few months ago). We would love to add more features, and mobile support would be nice. The old UI is certainly quite clunky and could use some work.

The highest priority in the webrepl space right now is to add webrepl client support to mpremote.
